src/sgame/sg_bot_nav.cpp

```diff
@@ -150,7 +150,11 @@ bool GoalInRange( const gentity_t *self, float r )
 
 	if ( self->botMind->goal.targetsCoordinates() )
 	{
-		return ( glm::distance( VEC2GLM( self->s.origin ), self->botMind->nav().glm_tpos() ) < r );
+		// We are a bit more lenient in the z direction here,
+		// making it count less so that if we are standing on
+		// a mgturret above our destination point, it still
+		// counts as a "we have reached our destination".
+		return ( Length2D( VEC2GLM( self->s.origin ) - self->botMind->nav().glm_tpos(), 0.2 ) < r );
```

Comment:
I feel like a constant amount of leeway in the z direction would work better than one scaling on the distance, because this will be too relaxed with a larger radius. For example when an alien seeks the booster it uses radius 200. 1000 units would be way too big.

Reply:
I agree in theory. In practice, however, the scaling trick seems to work fine.

Reply:
Fair enough. I've pushed a new version, how about this one ?

Reply:
LGTM